rustls/src/webpki/client_verifier.rs

```diff
@@ -247,13 +240,32 @@ pub struct WebPkiClientVerifier {
 
 impl WebPkiClientVerifier {
     /// Create builder to build up the `webpki` client certificate verifier configuration.
```

Comment:
I think now that there are two `builder`-ish functions this headline could use a tweak to emphasize that this one gives you some defaults like the ring crypto provider.

Reply:
Thanks, this is looking better I think -- 
![image](https://github.com/rustls/rustls/assets/579363/53137f06-cef8-4c08-8644-476c292a506b)

Reply:
Much better, thanks!